.taskcluster.yml

```diff
@@ -222,13 +222,11 @@ tasks:
                                     then: >
                                         cd /builds/worker/checkouts/src &&
                                         ln -s /builds/worker/artifacts artifacts &&
-                                        pip3 install -r requirements/base.txt &&
```

Comment:
One downside to switching to Poetry in this patch is that we can't pin the dependencies anymore. The `pip install .` command will resolve the latest dependencies.

We have a few options:
1. Leave it alone and deal with dependencies being unpinned.
2. Install poetry (either in cmd here or in Decision image) and use `poetry install` instead.
3. Continue generating `base.txt` via `poetry export`.

I like option 1 or 2.